src/ci/resources/cwl_conformance_test.wdl

```diff
@@ -63,10 +63,13 @@ task run_test_index {
         Int timeout
     }
 
+    # Weird -n/--timout format is because ./run_test.sh doesn't pass through the timeout parameter to cwltest.
```

Comment:
--timeout

Reply:
<sup>Hunh... this isn't the only `timout` in the code...</sup>

Reply:
I'm surprised / disappointed if `--timeout` wasn't being passed through. Are you sure it wasn't a `--timout`?